lib/pups/cli.rb

```diff
@@ -4,11 +4,16 @@
 
 module Pups
   class Cli
+    attr_accessor :ignored
+
     def self.opts
       OptionParser.new do |opts|
         opts.banner = 'Usage: pups [FILE|--stdin]'
         opts.on('--stdin', 'Read input from stdin.')
         opts.on('--quiet', "Don't print any logs.")
+        opts.on('--ignore <element(s)>', Array, "Ignore these template configuration elements, multiple elements can be provided (comma-delimited).") do |ignored|
+          @ignored = ignored
```

Comment:
`ignored` I just expect to be a boolean, maybe `ignored_elements` would be better

Reply:
funnily I had called it this originally, I'll change it back